hdx_hapi/endpoints/models/population.py

```diff
@@ -1,21 +1,35 @@
-from datetime import datetime
 from pydantic import ConfigDict, Field, NaiveDatetime
 from typing import Optional
 
-from hdx_hapi.endpoints.models.base import HapiBaseModel, HapiModelWithAdmins
+from hapi_schema.utils.enums import Gender
+from hdx_hapi.endpoints.models.base import HapiBaseModel
 
 
-class PopulationResponse(HapiBaseModel, HapiModelWithAdmins):
-    gender_code: Optional[str] = Field(max_length=1)
-    age_range_code: Optional[str] = Field(max_length=32)
+class PopulationResponse(HapiBaseModel):
```

Comment:
Could we use here `HapiModelWithAdmins` and remove the admin and location fields?

Reply:
I think this might be something I messed up resolving merge conflicts